django/test/client.py

```diff
@@ -1338,3 +1334,234 @@ async def request(self, **request):
         if response.cookies:
             self.cookies.update(response.cookies)
         return response
+
+    async def get(
```

Comment:
`follow` is not a part of the [connection scope](https://asgi.readthedocs.io/en/latest/specs/www.html#http-connection-scope) so I wonder if the current solution in the `AsyncRequestFactory.generic()` is correct :thinking: :
https://github.com/django/django/blob/76c3e310dd37a1d77642a8744db636a3a4337af2/django/test/client.py#L708-L710

Reply:
Check out https://github.com/django/django/pull/13635#discussion_r516601026. I think we should remove it and add an appropriate  regression test.

Reply:
`follow` cannot reach `AsyncRequestFactory.generic()` when we added `AsyncClient` methods for HTTP methods, so this code is unnecessary.

Reply:
@olibook Can you check my comments? :point_up: